package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "psi-cardinality",
+  "version": "1.0.0",
```

Comment:
I'd rather go for 0.0.1. A Version number >= 1 [implies a stable interface](https://en.wikipedia.org/wiki/Software_versioning#Change_significance), which I don't think we should promise yet.